### ISSUE TITLE
Warnings while generating docs

### ISSUE THREAD

**Issue by lukaszstolarczuk** [pmem/libpmemobj-cpp#352]
goto @igchor for more details

some warnings are thrown while generating documentation on 1.7 (using doxygen )

**Comment by lukaszstolarczuk** [comment]
There currently are some errors on CI (https://travis-ci.org/github/pmem/libpmemobj-cpp/jobs/706074355), but it should also be checked locally.

```
/libpmemobj-cpp/doc_update/libpmemobj-cpp/include/libpmemobj++/container/detail/concurrent_skip_list_impl.hpp:2142: warning: argument 'level' from the argument list of pmem::detail::concurrent_skip_list::internal_find_position has muliple @param documentation sections
/libpmemobj-cpp/doc_update/libpmemobj-cpp/include/libpmemobj++/container/detail/concurrent_skip_list_impl.hpp:2142: warning: argument 'key' from the argument list of pmem::detail::concurrent_skip_list::internal_find_position has muliple @param documentation sections
/libpmemobj-cpp/doc_update/libpmemobj-cpp/include/libpmemobj++/container/detail/concurrent_skip_list_impl.hpp:2142: warning: argument 'cmp' from the argument list of pmem::detail::concurrent_skip_list::internal_find_position has muliple @param documentation sections
```

**Comment by lukaszstolarczuk** [comment]
Fixed in #820
